contracts/BucketAuction.sol

```diff
@@ -32,7 +32,8 @@ contract BucketAuction is IBucketAuction, ERC721M {
             tokenURISuffix,
             maxMintableSupply,
             globalWalletLimit,
-            cosigner
+            cosigner,
+            300
```

Comment:
nit: add a parameter comment
```
/* timestampExpirySeconds= */ 300,
```

Also, is 5 min too long?

Reply:
sounds good will add, UX wise i think its fine and it can always be updated later